.github/scripts.js

```diff
@@ -0,0 +1,104 @@
+// Get every html element
+const searchInput = document.getElementById("searchInput");
+const searchButton = document.getElementById("searchButton");
+const searchResults = document.getElementById("searchResults");
+const theLoader = document.getElementById("loader");
+
+//create function to make my spinner visible.
+function enableSpinner() {
+  theLoader.classList.remove("invisible");
+}
+
+//create function to make my spinner unvisible
+function disableSpinner() {
+  theLoader.classList.add("invisible");
+}
+
+//create async function for my API, to display 10 results from the server and create the list.
+async function getSearchResults() {
+  const searchTerm = searchInput.value;
+  const endpoint = `https://stock-exchange-dot-full-stack-course-services.ew.r.appspot.com/api/v3/search?query=${searchTerm}&limit=10&exchange=NASDAQ`;
```

Comment:
store 'https://stock-exchange-dot-full-stack-course-services.ew.r.appspot.com/api/v3/' in baseUrl var, and use it wherever you need. DRY